PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -158,21 +184,30 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
         etaWidth = Var("superCluster().etaWidth()",float,doc="Width of the photon supercluster in eta", precision=8),
         phiWidth = Var("superCluster().phiWidth()",float,doc="Width of the photon supercluster in phi", precision=8),
         cutBased = Var(
-            "userInt('cutbasedID_loose')+userInt('cutbasedID_medium')+userInt('cutbasedID_tight')",
+            "userInt('cutBasedID_loose')+userInt('cutBasedID_medium')+userInt('cutBasedID_tight')",
+            int,
+            doc="cut-based ID bitmap, RunIIIWinter22V1, (0:fail, 1:loose, 2:medium, 3:tight)",
+        ),
+        cutBased_Fall17V2 = Var(
+            "userInt('cutbasedID_Fall17V2_loose')+userInt('cutbasedID_Fall17V2_medium')+userInt('cutbasedID_Fall17V2_tight')",
             int,
             doc="cut-based ID bitmap, Fall17V2, (0:fail, 1:loose, 2:medium, 3:tight)",
         ),
-        cutBased_Fall17V1Bitmap = Var(
-            "userInt('cutbasedIDV1_loose')+2*userInt('cutbasedIDV1_medium')+4*userInt('cutbasedIDV1_tight')",
+        cutBased_Fall17V1 = Var(
```

Comment:
For my curiosity, is the Fall17V1 ID even still supported? Or could it be removed?

Reply:
I have removed it

Reply:
Yes just to confirm, Fall17V1 is not supported anymore by EGM for Run2UL and for Run3.
Even for Run2 EOY ReReco, Fall17V1 is disfavoured. We have been encouraging analysers to move to Fall17V2 for years now. So I think it's a good time to remove Fall17V1 from nano.